index.html

```diff
@@ -396,21 +396,36 @@
           Install prompts
         </h2>
         <p>
-          An end-user can <dfn data-lt="manual installation">manually</dfn>
+          There are three ways that the installation process can be triggered:
```

Comment:
maybe change this to "multiple ways" instead of "three"... always forget to update those numbers when new things get added.